contracts/near/eth-prover/.catalog-info.yaml

```
@@ -0,0 +1,20 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: prover-on-near
+  description: Prover on NEAR
+  tags:
+    - contract
+    - near
+  links: []
+  annotations:
+    aurora.dev/security-tier: "1"
+spec:
+  owner: bridge-team
+  type: contract
+  lifecycle: production
+  system: bridge-protocol
+  deployedAt:
+    - contract:near/mainnet/prover.bridge.near
+  interactsWith: []
```

Prover interacts with Ethereum light client on NEAR

```suggestion
  interactsWith:
    - contract:near/mainnet/client.bridge.near
```